### PR TITLE
Some placeholder methods to cli

### PR DESCRIPTION
**Pull request by ankit8898**
Added a thor CLI with couple of methods

- clone_version
- update_version

```
~/D/p/p/akamai-cloudlet-updater> acm
Commands:
  acm clone_version                     # Clones the current version
  acm help [COMMAND]                    # Describe available commands or one specific command
  acm update_version --version=VERSION  # Updates the version

 ~/D/p/p/akamai-cloudlet-updater>

```

**update_version**
```
~/D/p/p/akamai-cloudlet-updater> acm help update_version
Usage:
  acm update_version --version=VERSION

Options:
  --version=VERSION

Description:
  Updates the version With --version, version which needs to be updated
 ~/D/p/p/akamai-cloudlet-updater>

```

**clone_version**
```
~/D/p/p/akamai-cloudlet-updater> acm help clone_version
Usage:
  acm clone_version

Description:
  Clones the current activated version. All new updates are performed in cloned
 ~/D/p/p/akamai-cloudlet-updater>
```

### CODE REVIEW

File: akamai_cloudlet_manager.gemspec [hunk 1]
```diff
@@ -13,12 +13,11 @@ Gem::Specification.new do |s|
...
                  ]
-  s.executables << 'ac'
+  s.executables << 'acm'
   s.homepage    = 'https://github.com/gouravtiwari/akamai-cloudlet-manager'
   s.license       = 'MIT'
   s.add_runtime_dependency 'akamai-edgegrid', '~> 1.0', '>= 1.0.6'
   s.add_runtime_dependency 'json', '~> 2.1'
   s.add_runtime_dependency 'thor', '~> 0.20.0'
   s.add_development_dependency 'rspec', '~> 3.7'
-  s.add_development_dependency 'rake'
```
Comment (gouravtiwari) [inline akamai_cloudlet_manager.gemspec]: I think rake is needed by travis, if we can keep it.

Comment (ankit8898) [reply akamai_cloudlet_manager.gemspec]: my bad.

File: bin/acm [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+require 'akamai_cloudlet_manager'
+require 'yaml'
+require 'thor'
+
+class AkamaiCloudletManagerCli < Thor
+
+  desc "clone_version", "Clones the current version"
+  long_desc <<-LONGDESC
+  Clones the current activated version. All new updates are performed in cloned
```
Comment (gouravtiwari) [inline bin/acm]: All new updates are performed in cloned _version_

File: Gemfile.lock [hunk 3]
```diff
@@ -35,7 +34,6 @@ PLATFORMS
 
 DEPENDENCIES
   akamai_cloudlet_manager!
-  rake
```
Comment (gouravtiwari) [inline Gemfile.lock]: Please revert this as well

**Pull request merged**